core-services/release-controller/_releases/priv/release-ocp-4.17-multi.json

```diff
@@ -212,6 +212,13 @@
             "prowJob": {
                 "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-heavy-build-ovn-remote-libvirt-multi-p-p-priv"
             }
+        },
+        "install-analysis-multi-p-p": {
```

Comment:
I don't think we're supposed to update the `/priv` directory files... Were these generated automatically?